.github/workflows/build-package-docs.yaml

```diff
@@ -0,0 +1,99 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
+        description: 'GitHub Fork'
+        required: true
+        default: 'ansible'
+      github_branch:
+        description: 'GitHub Branch'
+        required: true
+        default: 'devel'
+      language:
+        type: choice
+        description: 'Language'
+        required: true
+        default: 'english'
+        options:
+        - 'english'
+        - 'japanese'
+      package_version:
+        type: choice
+        description: 'Ansible Version'
+        required: true
+        default: 'devel'
+        options:
+        - 'devel'
+        - '9'
+        - '8'
+        - '7'
+        - '6'
+        - '5'
+        - '4'
+        - '3'
+        - '2.10'
+      latest_symlink:
+        type: boolean
+        description: 'Add latest symlink'
+        required: true
+
+jobs:
+  build-package-docs:
+    runs-on: ubuntu-latest
+    env:
+      PACKAGE_VERSION: "${{ github.event.inputs.package_version }}"
+      LANGUAGE: ${{ github.event.inputs.language}}
+    steps:
+      - name: Checkout Ansible documentation
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.inputs.github_fork }}/ansible-documentation
+          ref: ${{ github.event.inputs.github_branch }}
+          path: build-directory
+
+      - name: Setup nox
+        uses: wntrblm/nox@2024.03.02
+
+      - name: Output Python info
+        run: python --version --version && which python
+
+      - name: Graft ansible-core
+        run: nox -s clone-core
+        working-directory: build-directory
+
+      - name: Install project requirements
+        run: python -m pip install -r tests/requirements.in -c tests/requirements.txt
+        working-directory: build-directory
+
+      - name: Set the COLLECTION_LIST variable
+        run: |
+          if [ ${PACKAGE_VERSION} == "devel" ]; then
+            echo COLLECTION_LIST="" >> $GITHUB_ENV
```

Comment:
those double quotes won't actually end up recorded in `$GITHUB_ENV`. This will result in `COLLECTION_LIST=` because quotes are interpreted on the shell level and you record the output of the `echo COLLECTION_LIST=""`. If you try it out in your local terminal, you'll see what the actual output is.

Reply:
ah, nice catch! I'll ask before I commit but `echo COLLECTION_LIST="\"\""` is what we want here right?

Reply:
I don't know (does it work in your local terminal?)

Escaping quotes in bash/shell is tricky. But in this case, I'd go for wrapping the entire thing with single quotes. As in `echo COLLECTION_LIST='""'` or `echo 'COLLECTION_LIST=""'`.

OTOH, do you really want to set an empty var? It's probably a default anyway. You could invert the check to `!= "devel"` below and have less if/elif branches. Additionally, with just a single branch, you could even move the check to the GHA job step `if:` so that the entire step would be skipped if not needed and you wouldn't even need to have any conditional checks on the shell script side, leaving you with a one-liner.

Note that the same quoting concern applies to the non-empty variant of this variable, just a few lines below.

Reply:
Thanks again @webknjaz Yes, that worked locally. I prefer inverting the check to `!="devel"` that's a great suggestion.